crates/runtime/tests/models/mod.rs

```diff
@@ -217,17 +229,38 @@ pub fn get_mega_science_dataset(
             .into_iter()
             .collect(),
     ));
+
     dataset.acceleration = Some(Acceleration {
-        enabled: true,
+        enabled: accelerate,
         ..Default::default()
     });
 
-    dataset.columns = [question_column, answer_column]
+    dataset
+}
+
+// This dataset view is derived from https://huggingface.co/datasets/MegaScience/MegaScience, with the following alterations:
+//  - Any `question` or `answer` > 256 characters is removed.
+//  - An arbitrary but unique `id` integer column is added.
+pub fn get_mega_science_view(
+    spice_name: Option<&str>,
+    question_column: Option<spicepod::semantic::Column>,
+    answer_column: Option<spicepod::semantic::Column>,
+) -> (Dataset, View) {
+    let mut dataset = mega_science_dataset(Some("mega_science_ds"), false);
+
+    let mut v = View::new(spice_name.unwrap_or("megascience").to_string());
+    v.sql = Some("SELECT * FROM mega_science_ds".to_string());
```

Comment:
Maybe worth considering a slightly more complicated view?

Reply:
@Jeadie please test views with joins and unions (the two primary use-cases for views).